### PR TITLE
Fix of error handling (FB compatibility)

### PR DESCRIPTION
**Pull request by jendakol**

### CODE REVIEW

**Comment by jendakol** [comment]
cc @davidvavra @augard

File: internal/utils/http/http.go [hunk 1]
```diff
@@ -170,10 +170,8 @@ func SendErrorResponse(w http.ResponseWriter, r *http.Request, err error) {
 func sendErrorResponse(w http.ResponseWriter, r *http.Request, error rpccode.Code, message string) {
 	logger := logging.FromContext(r.Context())
 
-	errorCode := rpccode.Code_value[error.String()]
-
 	status := errorResponse{
-		Status:  errorCode,
+		Status:  error.String(),
```
Comment (davidvavra) [inline internal/utils/http/http.go:174]: Have you tested that it returns strings from column RPC here? https://cloud.google.com/apis/design/errors#handling_errors

Comment (jendakol) [reply internal/utils/http/http.go:174]: It returns these values: https://godoc.org/google.golang.org/genproto/googleapis/rpc/code
I didn't check it one by one but it looks the same, looking at it.

Comment (davidvavra) [reply internal/utils/http/http.go:174]: Looks good. Hopefully Google has same error codes in different producs :D

**Pull request merged**